help: Merge two similar sections in `marking messages as read`. (#24631)
zulip/zulip: help/marking-messages-as-read.md

## Configure whether messages are automatically marked as read

> You may want to configure the Zulip mobile app to automatically mark messages
> as read or not, depending on your preferences. For example, setting the app to
> never mark messages as read can be useful if you tend to follow up on the
> messages when you are on your computer. You will still be able to

Reviewer: How about something like the following for the intro:

```suggestion
You may want to configure whether the Zulip mobile app will automatically
mark messages as read. For example, it can be useful to set the app to never
mark messages as read  if you often want to follow up on messages when you
are at your computer. You will still be able to
```

Author: Hmm, yeah I like it, it flows better than what I had. Updating the PR.

Author: Actually would it be better if we made it "whether the Zulip mobile app will automatically mark messages as read or not." just to make it a bit more clear?

Author: That sounds good to me!
